_data/companies.yml

```
@@ -1062,6 +1062,13 @@
   events: "?"
   last_update: 2020-03-11
 
+- name: Optimizely
+  wfh: Mandatory
```

```suggestion
  wfh: Required
```
We're using required everywhere else, so I'm suggesting it here as well.